src/main/scala/nl.knaw.dans.easy.solr4files/Command.scala

```diff
@@ -21,10 +21,9 @@ import resource._
 
 import scala.language.reflectiveCalls
 import scala.util.control.NonFatal
-import scala.util.{ Failure, Success, Try }
+import scala.util.{ Failure, Try }
 
 object Command extends App with DebugEnhancedLogging {
-  type FeedBackMessage = String
 
   val configuration = Configuration()
   val commandLine: CommandLineOptions = new CommandLineOptions(args, configuration) {
```

Comment:
Is there any need not to do the `verify()` below inside the `CommandLineOptions`? Not sure about this. I need a reminder if you really need it this way, because I did it inside `CommandLineOptions` for most of the EASY-1291 issues.